src/components/teams/TeamsList.js

```diff
@@ -136,75 +123,117 @@ const TeamsList = () => {
-                            id="team-option-1-description">
-                            #{team.account_id}
-                          </p>
+                  <Fragment key={id}>
+                    <ConfigureHelpDesk
+                      isOpen={modalTeam === id}
+                      setIsOpen={() => {
+                        setModalTeam(null);
+                      }}
+                      team={team}
+                      helpDesk={firstHelpDesk}
+                    />
+                    <li className="py-5 flex flex-col items-center justify-between sm:py-8 sm:flex-row">
+                      <div className="flex w-full items-center space-x-5 sm:w-auto">
+                        <button
+                          onClick={() => goToAccessKey(team.id)}
+                          className="flex-shrink-0 flex items-center justify-center h-12 w-12 bg-purple-600 text-white text-sm font-medium rounded-lg">
+                          {team.name ? team.name.substring(0, 2) : "TL"}
+                        </button>
+                        <div className="flex flex-row space-x-16 items-center w-full justify-between sm:justify-start">
+                          <div className="flex flex-col sm:flex-row sm:items-center max-w-[10rem] sm:max-w-[8rem] md:max-w-none">
+                            <div className="flex flex-col">
+                              <p
+                                className="text-lg font-medium text-gray-900 leading-tight min-w-[6rem]"
+                                id="team-option-1-label">
+                                {team.name
+                                  ? team.name
+                                  : _x(
+                                      "Team {id}",
+                                      "{id} is replaced dynamically; do not translate",
+                                      "trustedlogin-connector"
+                                    ).replace("{id}", team.id)}
+                              </p>
+                              <p
+                                className="text-sm text-gray-500"
+                                id="team-option-1-description">
+                                #{team.account_id}
+                              </p>
+                            </div>
+                            {status === "error" && (
+                              <>
+                                <div
+                                  className="bg-orange-100 border-l-4 border-orange-500 text-orange-700 p-4 ml-4"
+                                  role="alert">
+                                  <p className="font-bold">
+                                    {__(
+                                      "Oops something went wrong",
+                                      "trustedlogin-connector"
+                                    )}
+                                    .
+                                  </p>
+                                  <p>{message}.</p>
```

Comment:
Let's move the periods to the message itself so we don't need to hunt down stray periods in the future.

Reply:
This is already addressed in the next PR.